h2/src/main/org/h2/value/ValueEnum.java

```diff
@@ -118,15 +118,15 @@ private static String sanitize(final String label) {
     }
 
     private static String toString(final String[] enumerators) {
-        String result = "(";
+        StringBuilder result = new StringBuilder("(");
         for (int i = 0; i < enumerators.length; i++) {
-            result += "'" + enumerators[i] + "'";
+            result.append("'").append(enumerators[i]).append("'");
             if (i < enumerators.length - 1) {
-                result += ", ";
+                result.append(", ");
             }
         }
-        result += ")";
-        return result;
+        result.append(")");
+        return result.toString();
```

Comment:
Usually `StatementBuilder` is used in H2 codebase for such operations, because in has `appendExceptFirst()` that may be used for `", "`.